hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This release expands the deprecation of timeout from `3.16.0 <_v3.16.0>` to
```

Comment:
Nitpick I just spotted in the changelog: this syntax doesn't cross-reference correctly.  Instead, you need

    :ref:`3.56.0 <v3.56.0>`

and it would be good to fix the same problem for 3.66.13 too.  (or deleting the underscore might be enough?)

Reply:
Ugh. Thanks. Fixed now.

I *really* wish we could get a better way of having sphinx tell us when something is wrong rather than just having it go "oh a syntax error. I bet they meant that to be plain text".